frontend/index.html

```diff
@@ -31,6 +31,13 @@ <h1 class="h3 mb-3 fw-normal">Welcome to the BOSAGORA Faucet</h1>
             <br>
               <input id="submitform" class="w-90 btn btn-lg btn-primary" type="submit" value="Send test BOA to address">
         </form>
+      <br>
+        <div id="valid" class="result" style="display: none;">
+          <p>Test BOA successfully sent!</p>
+        </div>
+        <div id="invalid" class="result" style="display: none;">
+          <p>Invalid address!</p>
```

Comment:
Here we are assuming the only error can be "invalid address".  What if the address is valid but it fails due to some other reason?

Reply:
Good point!